string operating_mode
string mir_paintbooth_location
---
bool response
